crates/ruff_python_formatter/README.md

```diff
@@ -0,0 +1,306 @@
+# Ruff Formatter
+
+The Ruff formatter is an extremely fast Python code formatter that ships as part of the `ruff`
+CLI (as of Ruff v0.0.287).
+
+The formatter is currently in an **alpha** state. As such, it's not yet recommended for production
+use, but it _is_ ready for experimentation and testing. _We'd love to have your feedback._
+
+## Goals
+
+The formatter is designed to be a drop-in replacement for [Black](https://github.com/psf/black),
+but with an excessive focus on performance and direct integration with Ruff.
+
+Specifically, the formatter is intended to emit near-identical output when run over Black-formatted
+code, achieving Jaccard similarity scores of over 0.999 on extensive Black-formatted projects like
```

Comment:
TODO: Add explanation link for Jaccard similarity

Reply:
i'm fine with stating this as ">99.9% of lines on projects [...] are formatted identical to black", the exact metric chosen doesn't correlate strongly with the user experience.